packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## v1.1.0-dev (TBD)
+- Required getToken for StorefrontPreview component [#1440]https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1440
 - Support Storefront Preview [#1413](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1413)
```

Comment:
All customers are going to care about is "Support Storefront Preview" - listing the changes from every individual PR isn't important (but you can still include multiple links for the "Support Storefront Preview" bullet point).

Reply:
I think the purpose of changelog is for us - developer keep track of our work. It is informative for customers, but it is technically not really useful for them. Release notes are likely what they care more.
Normally, when developers work on a codebase and have PR opened, they should also update changelog for the work they do.